zebrad/src/components/mempool/storage.rs

```diff
@@ -454,4 +466,21 @@ impl Storage {
         }
         Ok(())
     }
+
+    /// Update metrics related to the rejected lists.
+    ///
+    /// Must be called every time the rejected lists change.
+    fn update_rejected_metrics(&self) {
+        metrics::gauge!(
+            "mempool.rejected.transaction.ids.total",
```

Comment:
Naming Nitpick: I think we can just use `ids` as the unit for this gauge.

```suggestion
            "mempool.rejected.transaction.ids",
```
https://prometheus.io/docs/practices/naming/#metric-names

Reply:
Done in 660fb787